README.md

```diff
@@ -1,5 +1,59 @@
+## Introduction
 
+**What?** The [Neurodata Without Borders: Neurophysiology][nwb-neurophysiology] (NWB:N) Hackathon is a 3-day hands on
+activity in which neurophysiology researchers create solutions using the open source NWB:N software packages.
 
+Participants work collaboratively on solutions that use the NWB unified data format for cellular-based neurophysiology
+data, which is focused on the dynamics of groups of neurons measured under a large range of experimental conditions.
+In contrast to conferences and workshops where the primary focus is to report results, the objective of the Hackathon
+is to provide a venue for creators and users of neurophysiology open-source software to collaboratively work on any
+related research projects.
+
+[nwb-neurophysiology]: http://www.nwb.org/nwb-neurophysiology/
+
+**When, where, how much?** To be determined based on participation. This year there will be a development focused
+Hackathon in Early April in Seattle, and then a user on-boarding tutorial Hackathon in late April in Berkeley.
+Ad-hoc meetings are added occasionally. We plan to have at least one more Hackathon in 2019. For this first
+Hackathon, lunch and dinner will be covered and there will be no registration fee. These logistics are expected
+to evolve with the Hackathons. The [NAMIC Project Week][namic-and-3d-slicer-experience] which has influenced the
+NWB Hackathon currently has a registration fee that participants pay, which covers breakfast, lunch, coffee and
+snacks during the day.
+
+[namic-and-3d-slicer-experience]: http://www.spl.harvard.edu/publications/item/view/3004
+
+**How does it work?** Weekly videoconferences for preparation begin 8-12 weeks before an event. Potential participants
+propose projects during these meetings, and collaboratively create a list of projects that are of mutual interest. The
+projects include platform work, algorithm development, analytics, visualization, and neurophysiology applications and
+research tasks. Projects are not limited to code development, but can include creation of documentation and tutorial
+materials. Through the course of the meetings, each participant selects one or more project teams and develops goals
+for the Hackathon. The first day of the Hackathon itself begins with a 2-hour in-person introduction to all projects
+and participants. The rest of the Hackathon consists of a mix of working sessions and breakout sessions on special
+topics, as decided by the participants during the preparatory meetings.
+
+**Who can attend?** Hackathons are open to everyone and publicly advertised. One-day participation is permitted for
+first-time attendees, but participants are strongly encouraged to stay for the entire event. Email announcements
+are sent to the [Hackathon mailing list](To be done) (for continuity) and posted on
+the [Hackathon forum](https://github.com/orgs/NeurodataWithoutBorders/teams/hck04/discussions).
```

Comment:
@oruebel  @mgrauer  It would great to sort this out before inviting folks. Instead of the team https://github.com/orgs/NeurodataWithoutBorders/teams/hck04/discussions, slack would also work great during the hackathon. 

Let me know what you think ?

Reply:
I agree, it would be good to sort out single point of contact and communication channel, probably some other logistics also (will it cost anything?) before sending out the first message.

Reply:
For now, I believe both events are basically on invitation-only events. So for now I would say something along the lines of:

Due to limited space the hackathons participation is currently on an invitation-only basis. If you are interested in attending a hackathon then please contact the organizer of the hackathon directly. We are always excited to see new members join the NWB:N community.